blockly-core/tests/phantomwrapper.js

```diff
@@ -0,0 +1,76 @@
+var fs = require('fs');
+var WebPage = require('webpage');
+
+var DEFAULT_STYLE = "\033[39m";
+var GREEN = "\033[32m";
+var RED = "\033[31m";
```

Comment:
You could also use something like https://github.com/Marak/colors.js if you wanted.